pages/index.vue

```diff
@@ -40,6 +40,8 @@ export default {
   },
   mounted() {
     this.modifiedCanvas = this.$refs.modifiedCanvasEl;
+    this.modifiedCanvas.width = this.modifiedCanvas.clientWidth;
```

Comment:
Here, we set the Canvases data size to its physical size, so it must be drawn correctly.